### PR TITLE
Dependency updates and Travis CI

### PR DESCRIPTION
**Pull request by friederbluemle**
This updates all dependencies to the latest versions. Also included are some minor whitespace fixes and README updates.

I also enabled Travis CI and added a build status indicator to README.md (project needs to be [enabled on Travis](https://travis-ci.org/googlesamples/easypermissions) by an admin for this to work)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -5,10 +5,10 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.0'
+        classpath 'com.android.tools.build:gradle:2.1.3'
```
Comment (samtstern) [inline build.gradle]: Does it build with these deps for you?  I know we had an issue with `2.1.3` in the past.

**Comment by samtstern** [comment]
@friederbluemle thank you!  These are great changes, glad to have Travis coming online.

**Comment by samtstern** [comment]
Just checked and it builds!  Merging

**Comment by samtstern** [comment]
Build is green :-)
https://travis-ci.org/googlesamples/easypermissions/builds/159903204

**Comment by friederbluemle** [comment]
Great! :+1:

**Pull request merged**